src/components/DayTime.tsx

```diff
@@ -17,7 +17,7 @@ export default function DayTime(props: DayTimeProps): ReactElement {
     const [dayTime, setDayTime] = useState(getTimeOfDay(props.isStart));
     return (
         <StyledFormField>
-            <label>
+            <label style={{textAlign: 'left'}}>
```

Comment:
Styling should be separate from the HTML/JS. Create a styled component similar to `Span` below this function. Maybe call it `Label`.